src/sections/Meshmap/index.js

```diff
@@ -74,9 +74,9 @@ const Meshmap = () => {
                   <img src={mesheryCloud} className="meshery-hero-img" alt="Meshery Cloud" />
                   <div className="draft-logo">
                     <p>Help choose the best logo for MeshMap:</p>
-                    <a href="https://twitter.com/layer5/status/1536832439833743363?s=20&t=oEDasGEBl8cvsA64P7NNTA">
+                    <a target="_blank" href="https://twitter.com/layer5/status/1536832439833743363?s=20&t=oEDasGEBl8cvsA64P7NNTA">
                       <img src={MeshMapDrafts} alt="MeshMap Draft" /></a><br />
-                    <p><a href="https://twitter.com/layer5/status/1536832439833743363?s=20&t=oEDasGEBl8cvsA64P7NNTA">Vote for your favorite design</a></p>
+                    <p><a target="_blank" href="https://twitter.com/layer5/status/1536832439833743363?s=20&t=oEDasGEBl8cvsA64P7NNTA">Vote for your favorite design</a></p>
```

Comment:
Add `rel="noreferrer"` attribute

Reply:
Made the required changes in a new PR: #3308